package.json

```diff
@@ -22,5 +22,8 @@
   "homepage": "https://github.com/BlazarQSO/NodeJS#readme",
   "devDependencies": {
     "nodemon": "^3.0.1"
```

Comment:
as per AC1 and AC3:
The program should be started via npm script using nodemon

Reply:
Done.